libs/core/src/lib/form/form-message/form-message.component.scss

```diff
@@ -8,3 +8,7 @@
         max-width: 100%;
     }
 }
+
+.invisible {
```

Comment:
I think it should have some sort of prefix, o.w. can conflict with the same class in the application

Reply:
It's not used anywhere, so I will just remove